src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx

```diff
@@ -53,8 +53,12 @@ const HarvestAction: React.FunctionComponent<HarvestActionProps> = ({ pid, userD
   return (
     <ActionContainer>
       <ActionTitles>
-        <Title>CAKE </Title>
-        <Subtle>{t('Earned').toUpperCase()}</Subtle>
+        <Text bold textTransform="uppercase" color="secondary" fontSize="12px" pr="3px">
```

Comment:
Spacing need to be in units of 4px.

Reply:
Done

Reply:
I changed spacing in farm cards to 4px too